source/YearlyOverview/YearlyOverview.html

```diff
@@ -46,7 +52,9 @@ <h1 id="goalsHeader">Yearly Goals</h1>
                 <h1 id="monthHeader">Months</h1>
                 <div id="months">
                     <h2 id="January">
-                        <a href="../MonthlyOverview/MonthlyOverview.html">
+                        <a
```

Comment:
just like daily and monthly overview, I think it would be good to keep the ".." notation